Client/src/components/welcome.tsx

```diff
@@ -0,0 +1,16 @@
+import '../css/welcome.css';
+import { useNavigate } from 'react-router-dom';
+
+function Welcome() {
+  const navigate = useNavigate();
+  return (
+    <div className='welcome-page'>
+      <h1 className='welcome'>WELCOME.</h1>
+      <button className='options' onClick={() => navigate('/game')}>PLAY</button>
```

Comment:
Can these be h1 instead of buttons?

Reply:
Accessibility, at least somewhere in our site